test/rules/no-unbound-method/whitelist/test.tsx.lint

```diff
@@ -69,4 +69,12 @@ Validators.compose(Validators.required);
 (await someMethod())(c.method);
                      ~~~~~~~~ [0]
 
+const a = {
+  getText() {
+    return 'text';
+  }
+};
+
+delete a.getText;
```

Comment:
This test should also be in `test/rules/no-unbound-method/default/test.ts.lint` to make sure the option isn't enabled by default.

Reply:
I have added the test to the default test suite